benchmarks/dynamo/common.py

```diff
@@ -58,6 +58,9 @@
 # We are primarily interested in TF32
 torch.backends.cuda.matmul.allow_tf32 = True
 
+# Suppress torch.profiler spam
+os.environ["KINETO_LOG_LEVEL"] = "5"
```

Comment:
n00b question, how do you know kineto log level 5 can suppress the logs mentioned in the PR summary?

Reply:
nvm, some fbgs/fbgf gives me the answer.